docs/shared/filter_topo.yaml

```diff
@@ -1,6 +1,8 @@
 filter_topo:
   config:
+    filtered_orog: C403_filtered_orog.tile7.nc
```

Comment:
See comment above about this: If this is a magic value that cannot be changed, I think maybe it shouldn't appear in config.